packages/domain-search/src/components/suggestion-cta/index.tsx

```diff
@@ -34,15 +35,40 @@ export const DomainSuggestionCTA = ( { domainName }: DomainSuggestionCTAProps )
 	} = useMutation( {
 		mutationFn: async ( { acceptedTrademarkClaim }: { acceptedTrademarkClaim: boolean } ) => {
 			if ( acceptedTrademarkClaim ) {
-				return cart.onAddItem( suggestion );
+				const result = cart.onAddItem( suggestion );
+				events.onAddDomainToCart(
+					domainName,
+					suggestion.position,
+					suggestion.is_premium || false,
```

Comment:
```suggestion
					suggestion.is_premium ?? false,
```

Let's prefer null coalescing to keep consistency.